cmd/src/lsif_upload.go

```diff
@@ -208,7 +210,17 @@ Examples:
 			return fmt.Errorf("error: %s\n\n%s", resp.Status, body)
 		}
 
-		fmt.Printf("LSIF dump uploaded.\n")
+		payload := struct {
+			ID string `json:"id"`
+		}{}
+		if err := json.Unmarshal(body, &payload); err != nil {
+			return err
+		}
+
+		jobURL := string(base64.URLEncoding.EncodeToString([]byte(fmt.Sprintf(`LSIFJob:"%s"`, payload.ID))))
+		fmt.Println("")
+		fmt.Printf("LSIF dump successfully uploaded. It will be converted asynchronously.\n")
+		fmt.Printf("To check the status of the conversion, visit %s/site-admin/lsif-jobs/%s.\n", cfg.Endpoint, jobURL)
```

Comment:
```suggestion
		fmt.Printf("To check the status, visit %s/site-admin/lsif-jobs/%s.\n", cfg.Endpoint, jobURL)
```

I removed the term "conversion" because users probably won't know what that means (converted to what?).